### DISCUSSION TITLE
[UI idea] Make Hedy web page installable as a progressive web app (PWA)

### DISCUSSION THREAD

**Discussion by julianfoad** [hedyorg/hedy#4660] [Large Projects & Plans]
**Idea**
On a mobile phone I would like to be able to "install" the web page as a PWA.

Why? When running Hedy this way, more of the phone screen is then available (less of the browser gets in the way) and the user can't so easily switch out of it by accident.

I would visit Hedy in a browser first, and there use the browser's "install" option, by which the browser would make a Hedy launcher icon on the phone's home screen. Launching this would load Hedy in a single web-app view, with some of the browser's chrome hidden (that is, the browser's address bar, options, tab switcher etc.) 

How: guide and reference is available at https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps

(The principle works for larger devices as well as phones, but the advantages are particularly significant on small screens.)

**Comment by hasan-sh** [comment]
Amazing!! Enabling our website with PWA is definitely a useful addition. One other advantage is push notification providing  a great UX, which is a must moving forward.

**Comment by Felienne** [comment]
Lovely idea! I am thinking this could be a fun project for am undergrad student, since the scope is very clear and it is not really in the main path of Hedy. I will move this to Discussions/Large project plans & ideas (which is where I send students)
